scripts/bumpDocsVersion.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * This script is run after a new release of rsuite has been published,
+ * in order to bump the rsuite dependency in `docs/` folder to the latest version.
+ */
+const path = require('path');
+const { spawnSync } = require('child_process');
+const PackageJson = require('@npmcli/package-json');
+const projectRoot = process.cwd();
+
+(async () => {
+  const rsuitePackageJson = await PackageJson.load(projectRoot);
+  const docsPackageJson = await PackageJson.load(path.join(projectRoot, 'docs'));
+
+  const rsuiteVersion = rsuitePackageJson.content.version;
+
+  console.info('Local rsuite version is: ' + rsuiteVersion);
+
+  // Two steps to bump rsuite in docs
+  // 1. update docs version to the same as the latest rsuite version
+  // 2. install latest rsuite version in docs
+  console.log();
```

Comment:
Why is there an empty console.log output

Reply:
To add an empty line before the message for better output formatting.